Core/Src/main.c

```diff
@@ -139,6 +142,8 @@ int main(void)
   MX_FATFS_Init();
   /* USER CODE BEGIN 2 */
 
+  IMU_Init();
```

Comment:
You can probably leave this out of main.c for now; this PR is for IMU drivers so any initialization sequences can be referenced from the test file.

Reply:
left comments for todo